### PR TITLE
chore: bump oxc 0.11.0

### PR DESCRIPTION
**Pull request by underfin**
### Test Plan

---

### CODE REVIEW

File: crates/rolldown_resolver/Cargo.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ repository.workspace = true
 workspace = true
 
 [dependencies]
-oxc_resolver   = { workspace = true }
+oxc_resolver   = { workspace = true, features = ['package_json_raw_json_api'] }
```
Comment (Boshen) [inline crates/rolldown_resolver/Cargo.toml]: What's needed from `package_json_raw_json_api`? Let's try and avoid this for performance reasons.

File: crates/rolldown_error/Cargo.toml [hunk 2]
```diff
@@ -16,5 +16,5 @@ workspace = true
 [dependencies]
 ariadne    = { workspace = true }
 napi       = { workspace = true, optional = true }
-oxc        = { workspace = true }
+oxc        = { workspace = true, features = ["sourcemap"] }
```
Comment (Boshen) [inline crates/rolldown_error/Cargo.toml]: You should enable the feature on the workspace level

Comment (underfin) [reply crates/rolldown_error/Cargo.toml]: Done

File: crates/rolldown_resolver/Cargo.toml (hunk 1 above)
Comment (underfin) [reply crates/rolldown_resolver/Cargo.toml]: I upgrade the version, already fix it.

**Comment by underfin** [comment]
Look like the js side ci time is not correct at now. I run threejs10x at local using debug build, the pr is 800ms, the main branch is 1.6s.

**Comment by hyf0** [comment]
> Look like the js side ci time is not correct at now. I run threejs10x at local using debug build, the pr is 800ms, the main branch is 1.6s.

Prabably due to https://github.com/rolldown/rolldown/pull/699#issuecomment-2028027139.

**Comment by underfin** [comment]
Could you add the pervious rust benchmark to ci. Let us see the actual performance.

**Comment by hyf0** [comment]
> Could you add the pervious rust benchmark to ci. Let us see the actual performance.

I'm fixing the current one. It should shows the actual performance after being fixed.

**Review by hyf0 — APPROVED** [review]

**Pull request merged**